static/js/pages/committee-single.js

```diff
@@ -356,6 +362,19 @@ $(document).ready(function() {
         })
       );
       break;
+    case 'independent-expenditure-committee':
+      path = ['committee', committeeId, 'schedules', 'schedule_e', 'by_candidate'];
+      tables.DataTable.defer($table, {
+        path: path,
+        query: query,
+        columns: expendituresColumns,
+        // Order by receipt date descending
```

Comment:
Super small thing, but we should remove this comment because it's not actually what's happening.